components/ImageR/imageR.css

```diff
@@ -0,0 +1,21 @@
+.square-image {
+    height: 9rem;
+    width: 9rem;
+    margin: 10px;
+    object-fit: cover;
+}
+
+.round-image {
+    height: 9.5rem;
+    width: 9.5rem;
+    border-radius: 50%;
```

Comment:
Do pass this value of 50% as a variable in your variables.css file with --border-rd-circle:50% for better reading practises in code

Reply:
done